src/esp/gfx/GenericDrawable.cpp

```diff
@@ -183,6 +187,39 @@ void GenericDrawable::draw(const Mn::Matrix4& transformationMatrix,
     shader_->bindObjectIdTexture(*(materialData_->objectIdTexture));
   }
 
+  if (skinData_) {
+    // Gather joint transformations
+    auto& skin = skinData_->skinData->skin;
+    auto& jointIdToArticulatedObjectNodes =
+        skinData_->jointIdToArticulatedObjectNode;
+    auto& transformNodes = skinData_->jointIdToTransformNode;
+    Cr::Containers::Array<Mn::Matrix4> jointTransformations{
```

Comment:
This looks like dynamic allocation. Can we just do this once when creating the GenericDrawable?